### PR TITLE
Add i18n to create-profile, developers and post-job components. Remove companies i18n references.

### PR DESCRIPTION
**Pull request by PBillingsby**
## OVERVIEW

The following components have been mostly internationalized passing in dummy values to interpolations as placeholders for future database values.

`create-profile`
`developers`
`post-job`

### CODE REVIEW

File: client/components/companies/CompanyItem.tsx [hunk 1]
```diff
@@ -41,9 +44,15 @@ export default function CompanyItem() {
...
-      <Text as="i">Open Jobs: 10</Text>
+      <Text as="i">
+        {t('components.companies.company_item.open_jobs', {
+          count: 1 /* Placeholder count */,
+        })}
+      </Text>
       <br />
-      <ButtonGray>View More</ButtonGray>
+      <ButtonGray>
+        {t('components.companies.company_item.button_text')}
```
Comment (frankTurtle) [inline client/components/companies/CompanyItem.tsx:54]: Could you help me understand where it pulls the string from this to translate it? I can't seem to find the connection from `'components.companies.company_item.button_text` -> "View More"

Comment (frankTurtle) [reply client/components/companies/CompanyItem.tsx:54]: Scratch that. I found it at the bottom of the PR 🤙🤙

**Review by frankTurtle — APPROVED** [review]
Looks solid to me 🤙

**Comment by angeljgomezc** [comment]
Hey @PBillingsby! Deleted companies components because we aren't working with those in the MVP. We'll add them later, so you may need to fix your PR accordingly :)

**Comment by PBillingsby** [comment]
@angeljgomezc Done.

**Review by carlomigueldy — APPROVED** [review]
Looks very neat boss @PBillingsby and this is something we're happy to add into codebase!! Thanks for the initiative mate and the time spent adding this i18n 🙌

**Comment by carlomigueldy** [comment]
@allcontributors please add @PBillingsby for code, translation

**Comment by carlomigueldy** [comment]
@allcontributors please add @kasthor for code, translation

**Pull request merged**